### ISSUE TITLE
minetest.wrap_text equivalent?

### ISSUE THREAD

**Issue by Tmanyo** [minetest/minetest#6033]
Whenever I try to use minetest.wrap_text it does not seem to work.  It always says that I am attempting to call field 'wrap_text' (a nil value).  So I wanted to wrap my text in a formspec that I am making today and decided to write my own function.  It seems to do what minetest.wrap_text(str, limit) should do and it consumes quite a few less lines in the lua file.

**Comment by octacian** [comment]
`minetest.wrap_text` is not a part of the API. It is not documented, nor has it every been to my knowledge.

**Comment by sfan5** [comment]
it is part of the api: see [here](https://github.com/minetest/minetest/blob/2652d8db19fb2758de1da4c04ce169f80c87e5c7/doc/lua_api.txt#L2119)

which Minetest version are you using?

**Comment by octacian** [comment]
huh, Must've been added recently. Gotta remember that rubenwardy's nice formatted API isn't always the most up-to-date. Whoops. However, according to git log, this feature wasn't really added all that recently that I see O_o. Very unclear to me, sorry.

**Comment by Tmanyo** [comment]
I have tried using it before even back in 0.4.13 I think is the first time I tried using it.  It doesn't work on any version from what I can tell.

**Comment by Wuzzy2** [comment]
I have tested in 0.4.16, and can confirm. I used `luacmd` to test.

Apparently the function does not obey its documentation and returns something different than promised.

Actually, it returns a *table* of strings. As if the source string has been splitted. There are no newlines. This doesn't look useful.

To test for yourself, just try to call `minetest.wrap_text` with help of the `luacmd` and experiment a bit. You will see what happens.
